src/mkbktr.cpp

```diff
@@ -258,13 +258,13 @@ int main(int argc, char **argv) {
   // Get pointers to the header areas
   NcaFsHeader *fs_headers[4] = {
       reinterpret_cast<NcaFsHeader *>(
-          &patch_header_plaintext[0x400 + 0x200 * 0]),
+          &patch_header_plaintext[0x400 + (0x200 * 0)]),
       reinterpret_cast<NcaFsHeader *>(
-          &patch_header_plaintext[0x400 + 0x200 * 1]),
+          &patch_header_plaintext[0x400 + (0x200 * 1)]),
       reinterpret_cast<NcaFsHeader *>(
-          &patch_header_plaintext[0x400 + 0x200 * 2]),
+          &patch_header_plaintext[0x400 + (0x200 * 2)]),
```

Comment:
at first i thought this was a buffer over-run before i remembered PEMDAS, so I just tried to make it more obvious for idiots of my caliber